mule-apikit-metadata/src/test/resources/org/mule/module/apikit/metadata/headers-in-raml10/get-headers-router-config.out

```diff
@@ -0,0 +1,35 @@
+%type _:Java = (inputMetadata:{
+  "message" : @typeId("value" : "org.mule.runtime.api.message.Message") {
+    "payload" : Any, 
+    "attributes" : {
+      "queryParameters"? : {
+
+      }, 
+      "headers"? : {
+        "withoutType"? : String, 
+        "integerHeader"? : @range("from" : 10.0, "to" : 20.0) @int Number, 
```

Comment:
In the case of RAML 0.8, the metadata generated was "@int @range("from" : 10, "to" : 20) @default("value" : "15") Number".

default value is missing, is it right?

Reply:
The same behavior is present for all types, except for string

Reply:
This is the behavior defined in metadata-model-raml metadata converter.